src/test/java/seedu/address/testutil/PropertyBuilder.java

```diff
@@ -14,11 +14,11 @@
  */
 public class PropertyBuilder {
 
-    public static final String DEFAULT_NAME = "Mayfair";
-    public static final String DEFAULT_TYPE = "condo";
-    public static final String DEFAULT_ADDRESS = "1 Jurong East Street 32, #08-111";
-    public static final String DEFAULT_POSTAL = "609477";
-    public static final LocalDate DEFAULT_DEADLINE = LocalDate.parse("2021-12-31");
+    public static final String DEFAULT_NAME = "Block 123";
+    public static final String DEFAULT_TYPE = "HDB";
+    public static final String DEFAULT_ADDRESS = "456 Chua Chu Kang Street 88, #06-123";
+    public static final String DEFAULT_POSTAL = "609456";
+    public static final LocalDate DEFAULT_DEADLINE = LocalDate.parse("2021-08-23");
```

Comment:
Any reason for this change?

Reply:
you could actually just add your new property details to TypicalProperties file under testutil rather than editing the default values of property builder

Reply:
Thats true, would changing the default values of property builder be alright though, then I wouldn't have to edit again

Reply:
haha think it shouldn't matter but for future reference I think u should just add it to the TypicalProperties file ba 👍

Reply:
noted, though i should warn you, i would probably be doing the same thing for appointment to keep it consistent XD